<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alphuntue</class>
 <widget class="QMainWindow" name="alphuntue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>alphuntue</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="GSL_tab">
     <attribute name="title">
      <string>General Speakers List</string>
     </attribute>
     <widget class="QComboBox" name="GSL_GSLcountry">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>70</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::InsertAlphabetically</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="GSL_addCountry">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QListWidget" name="GSL">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>291</width>
        <height>281</height>
       </rect>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="movement">
       <enum>QListView::Snap</enum>
      </property>
     </widget>
     <widget class="QLabel" name="GSL_speakingCountry">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>GSL_speakingCountry</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="GSL_nextSpeaker">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next Speaker</string>
      </property>
     </widget>
     <widget class="QLabel" name="GSL_countryImage">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string>GSL_countryImage</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="GSL_timeBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>421</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>501</x>
        <y>31</y>
        <width>182</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="GSL_label">
         <property name="text">
          <string>Speaker's time:</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="GSL_time_min">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="GSL_time_s">
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="GSL_yield">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Yield to...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GSL_resume">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Resume</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GSL_stop">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GSL_pause">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="GSL_elapsedTime">
         <property name="text">
          <string>00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="GSL_timeSeperator">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="GSL_totalTime">
         <property name="text">
          <string>01:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="SS">
     <attribute name="title">
      <string>Single Speaker</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="unmod_tab">
     <attribute name="title">
      <string>Unmoderated Caucus</string>
     </attribute>
     <widget class="QProgressBar" name="unmod_progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>741</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="unmod_topic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>731</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>unmod_topic</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>203</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="unmod_hour">
         <property name="suffix">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="unmod_minute">
         <property name="suffix">
          <string>min</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="unmod_second">
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>731</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Topic:</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="unmod_topicInput"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>216</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>64</number>
       </property>
       <item>
        <widget class="QPushButton" name="unmod_start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="unmod_stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>106</width>
        <height>15</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="unmod_elapsedTime">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="unmod_totalTime">
         <property name="text">
          <string>00:00:00</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>unmod_progressBar</zorder>
     <zorder>unmod_topic</zorder>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Moderated Caucus</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCountries_present"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCountries_present">
   <property name="text">
    <string>Countries present</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
